mii/batching/ragged_batching.py

```diff
@@ -147,6 +147,10 @@ class RaggedRequest:
     _generated_tokens: List[torch.Tensor] = field(default_factory=list)
     _finish_reason: GenerationFinishReason = GenerationFinishReason.NONE
 
+    @property
+    def prompt_length(self) -> int:
```

Comment:
In the case we have deadlock, the `prompt_length` will be incorrect on the output. Perhaps we need to store `original_prompt_length` or something like that. Don't worry about this for now. I have plans to expand the output details and can handle this in a future PR.

Reply:
I see, you are right. I would appreciate it if you could fix it on your PR